launch/sim.launch.py

```diff
@@ -11,32 +11,27 @@
     SetEnvironmentVariable,
     Shutdown,
 )
-from launch.conditions import IfCondition
 from launch.launch_description_sources import PythonLaunchDescriptionSource
-from launch.substitutions import LaunchConfiguration, PythonExpression
+from launch.substitutions import LaunchConfiguration
 
 
 def generate_launch_description():
     bringup_dir = Path(get_package_share_directory("rj_robocup"))
     launch_dir = bringup_dir / "launch"
 
+    config_yaml = LaunchConfiguration("config_yaml", default="sim.yaml")  # noqa: F841
+    use_internal_ref = LaunchConfiguration(  # noqa: F841
+        "use_internal_ref", default="True"
+    )
+    use_sim_radio = LaunchConfiguration("use_sim_radio", default="True")  # noqa: F841
     team_flag = LaunchConfiguration("team_flag", default="-b")
-    use_grsim = LaunchConfiguration("use_grsim", default="True")
     ref_flag = LaunchConfiguration("ref_flag", default="-noref")
-    headless_flag = LaunchConfiguration("headless_flag", default="")
+    direction_flag = LaunchConfiguration("direction_flag", default="plus")  # noqa: F841
```

Comment:
it would be nice if this could be translated to its semantic meaning, e.g.

`team_color_flag, default='yellow'`

which is then translated to the right direction. A checkbox in the UI would actually be fully ideal here.

Reply:
(but you don't have to do the UI thing, we can bundle that with other UI changes)

Reply:
We can do this with ui changes in the summer if that's ok with you